candies/cli/parsers/standard.py

```diff
@@ -76,15 +77,65 @@ def configure(parser: ArgumentParser, command: Command, prefix: str = '.'):
         {'.command': 'tool', '..command': 'install', 'g': 'something'}.
     """
 
+    def is_(x: inspect.Parameter, of: type) -> bool:
+        return isinstance(x, of) or x == of
+
+    def configure_arg(x: inspect.Parameter, config: Dict):
+        parser.add_argument(parameter.name,
+                            **config)
+
+    def configure_flag(x: inspect.Parameter, config: Dict):
+        # This actually makes an arg to behave like a flag,
+        # so one could call `--x` instead of `--x y`.
+        config['action'] = 'store_true'
+
+        if parameter.default is parameter.empty:
+            config['default'] = False
+        else:
+            config['default'] = bool(parameter.default)
+
+        short = getattr(parameter.annotation, 'short', None)
+        short = short if short is not None else parameter.name[0]
+
+        if short is '':
+            raise ValueError(f"Flag '{parameter.name}' has invalid short name "
+                             f"'{short}': "
+                             f"it is an empty string")
+
+        if len(short) > 1:
+            raise ValueError(f"Flag '{parameter.name}' has invalid short name "
+                             f"'{short}': "
+                             f"it consists of more than one character")
+
+
+        if not short.isalpha():
+            raise ValueError(f"Flag '{parameter.name}' has invalid short name "
+                             f"'{short}': "
+                             f"it is not an alphabetic letter")
+
+        parser.add_argument(f'--{parameter.name}',
+                            f'-{short}',
```

Comment:
Should we consider other prefixes? I believe they should be configurable.

Reply:
We definitely should. I only remember that `+` is sometimes used, but not sure this is the only one option: `/` is sometimes used too. Created #19 for that case. Thanks!